aeron-driver/src/main/c/aeron_publication_image.h

```diff
@@ -313,7 +313,13 @@ inline int64_t aeron_publication_image_join_position(aeron_publication_image_t *
 
     for (size_t i = 0, length = image->conductor_fields.subscribable.length; i < length; i++)
     {
-        int64_t sub_pos = aeron_counter_get_volatile(image->conductor_fields.subscribable.array[i].value_addr);
+        aeron_tetherable_position_t *tetherable_position = &image->conductor_fields.subscribable.array[i];
+        if (AERON_SUBSCRIPTION_TETHER_RESTING == tetherable_position->state)
+        {
+            continue;
+        }
+
+        int64_t sub_pos = aeron_counter_get_volatile(tetherable_position->value_addr);
```

Comment:
This is not quite correct either ;-) We should only consider for join position when in the ACTIVE state, like the Java driver does.

Reply:
Ah I didn't know the Java driver already did this correctly. Fixed ( I think for real this time :) )